src/SearchInput.tsx

```diff
@@ -0,0 +1,31 @@
+import { css } from "emotion";
+import React, { ChangeEvent } from "react";
+
+const inputCss = css`
+  justify-self: flex-end;
+  width: 200px;
```

Comment:
We should avoid hardcoding all of these widths and heights. Of the 30k users this site has got, 90% are mobile and this search bar looks a little off on mobile